### PR TITLE
Add ban startup

### PR DESCRIPTION
**Pull request by MattiSG**
@yohanboniface Please review ;)
Logo URL in particular.

### CODE REVIEW

**Comment by MattiSG** [comment]
@cquest

File: _startups/ban.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: Base Adresse Nationale
+mission: "Un référentiel national ouvert : de l'adresse à la coordonnée géographique."
+domain: adresse.data.gouv.fr
+repository: https://github.com/etalab/ban
+status: consolidation
```
Comment (yohanboniface) [inline _startups/ban.md]: Not sure about the "consolidation" status, what are the choices?

**Comment by yohanboniface** [comment]
I'll let @cquest deal with the logo.

File: _startups/ban.md (hunk 1 above)
Comment (MattiSG) [inline _startups/ban.md]: https://github.com/sgmap/sgmap.github.io/blob/master/_config.yml#L29-L52

(and https://github.com/sgmap/sgmap.github.io/blob/master/CONTRIBUTING.md)

File: _startups/ban.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Base Adresse Nationale
+mission: "Un référentiel national ouvert : de l'adresse à la coordonnée géographique."
+domain: adresse.data.gouv.fr
+repository: https://github.com/etalab/ban
```
Comment (MattiSG) [inline _startups/ban.md]: Is it the proper one? Not listed in http://adresse.data.gouv.fr/foss/

Comment (yohanboniface) [inline _startups/ban.md]: Depends who is targeted: https://github.com/etalab/ban is where the work currently focuses (CRUD API to make it simple), and that *will* be the actual **B** of BAN,  but it's not yet in production. So if we want to say "here is where the team is working", that's OK. If we want to say "here is the code actually released", better to use https://github.com/etalab/adresse.data.gouv.fr as an entry point I'd say.

File: _startups/ban.md (hunk 1 above)
Comment (yohanboniface) [inline _startups/ban.md]: So I think "construction" is better suited.

Comment (MattiSG) [inline _startups/ban.md]: :+1:

File: _startups/ban.md (hunk 2 above)
Comment (MattiSG) [inline _startups/ban.md]: Ok. Considering the lack of a README on `etalab/adresse.data.gouv.fr`, I think the best current destination would be http://adresse.data.gouv.fr/foss/. WDYT?

Comment (yohanboniface) [inline _startups/ban.md]: Yes, good suggestion, let's use that one! :)

**Pull request merged**